learn/data_backup/dumps.mdx

```diff
@@ -1,23 +1,31 @@
 ---
-title: Dumps — Meilisearch documentation
+title: Exporting and importing dumps — Meilisearch documentation
 description: Dumps are data backups containing all data related to a Meilisearch instance. They are often useful when migrating to a new Meilisearch release.
 ---
 
-# Dumps
+# Exporting and importing dumps
 
-Meilisearch stores its database in files located in `./data.ms` by default. Because this database is bound to the version of Meilisearch that created it, you must use dumps to migrate data between Meilisearch releases.
+A [dump](/learn/advanced/snapshots_vs_dumps#dumps) is a compressed file containing an export of your Meilisearch instance. Use dumps to migrate to new Meilisearch versions. This tutorial shows you how to create and import dumps. 
 
-A dump is a compressed file containing an export of your Meilisearch instance. It contains all your indexes, documents, and settings, but in a raw unprocessed form. A dump isn't an exact copy of your database—it is closer to a blueprint that allows you to create an identical dataset.
-
-Creating a dump is also referred to as exporting it, whereas launching Meilisearch with a dump is referred to as importing it.
+Creating a dump is also referred to as exporting it. Launching Meilisearch with a dump is referred to as importing it.
 
 ## Creating a dump
 
-To create a dump of your dataset, use the [create a dump endpoint](/reference/api/dump#create-a-dump):
+### Creating a dump in Meilisearch Cloud
+
+**You cannot manually export dumps in Meilisearch Cloud**. To migrate your project to the most recent Meilisearch release, use the Cloud interface:  
+
+![The General settings interface displaying various data fields relating to a Meilisearch Cloud project. One of them reads "Meilisearch version". Its value is `v1.6.2`. Next to the value is a button "Update to v1.7.0"](https://raw.githubusercontent.com/meilisearch/documentation/main/assets/images/cloud-dumps/01-import-dump.png)
+
+If you need to create a dump for reasons other than upgrading, contact the support team via the Meilisearch Cloud interface or the [official Meilisearch Discord server](https://discord.meilisearch.com).
+
+### Creating a dump in a self-hosted instance
+
+To create a dump, use the [create a dump endpoint](/reference/api/dump#create-a-dump):
 
 <CodeSamples id="post_dump_1" />
 
-The above code triggers a dump creation process. It also returns a [summarized task object](/learn/async/asynchronous_operations#summarized-task-objects) that you can use to check the status of your dump.
+This will returns a [summarized task object](/learn/async/asynchronous_operations#summarized-task-objects) that you can use to check the status of your dump.
```

Comment:
```suggestion
This will return a [summarized task object](/learn/async/asynchronous_operations#summarized-task-objects) that you can use to check the status of your dump.
```